client/snippets/snippets.json

```diff
@@ -39,5 +52,267 @@
+        "description": "Recipient verb with adjective"
+    },
+    "Sender Action": {
+        "prefix": "\\senderAction",
+        "body": "\\senderAction(VERB_${1|EN,FR,DE,ES,JA,NL,PT|}_${2:VERBNAME}, ${3|PRESENT_INDICATIVE,AUX_MOD,AUX_MOD_CONTINUOUS,AUX_MOD_PAST_PERFECT,AUX_MOD_PAST_PERFECT_CONTINUOUS,AUX_MOD_PERFECT,AUX_MOD_PERFECT_CONTINUOUS,AUX_MOD_PRETERIT,AUX_MOD_PRETERIT_CONTINUOUS,CONDITIONAL,CONJUNCTIVE_ROOT,CONTINUOUS_FUTURE,CONTINUOUS_FUTURE_PERFECT,CONTINUOUS_PAST_CONDITIONAL,CONTINUOUS_PAST_PERFECT,CONTINUOUS_PRESENT,CONTINUOUS_PRESENT_CONDITIONAL,CONTINUOUS_PRESENT_PERFECT,CONTINUOUS_PRETERIT,FUTURE,FUTURE_1_INDICATIVE,FUTURE_2_INDICATIVE,FUTURE_INDICATIVE,FUTURE_PERFECT,FUTURE_PERFECT_INDICATIVE,FUTURE_PERFECT_SUBJUNCTIVE,FUTURE_SUBJUNCTIVE,FUTURE_SUBJUNCTIVE_1,FUTURE_SUBJUNCTIVE_2,GERUNDIVE,IMPERATIVE,IMPERFECT_INDICATIVE,IMPERFECT_SUBJUNCTIVE,INACCOMPLISHED_ROOT,INFINITIVE,INFINITIVE_1,INFINITIVE_2,PARTICIPLE,PARTICIPLE_1,PARTICIPLE_2,PARTICIPLE_3,PAST,PAST_CONDITIONAL,PAST_CONDITIONAL_SECOND_FORM,PAST_GERUNDIVE,PAST_HISTORIC_INDICATIVE,PAST_IMPERATIVE,PAST_INFINITIVE,PAST_PARTICIPLE,PAST_PERFECT,PAST_PERFECT_INDICATIVE,PAST_SUBJUNCTIVE,PAST_SUBJUNCTIVE_1,PAST_SUBJUNCTIVE_2,PERFECT_INDICATIVE,PLU_PERFECT_INDICATIVE,PLU_PERFECT_SUBJUNCTIVE,POTENTIAL,PRESENT,PRESENT_CONDITIONAL,PRESENT_IMPERATIVE,PRESENT_INFINITIVE,PRESENT_OR_FUTURE,PRESENT_PARTICIPLE,PRESENT_PERFECT,PRESENT_PERFECT_INDICATIVE,PRESENT_SUBJONCTIVE_1,PRESENT_SUBJONCTIVE_2,PRESENT_SUBJUNCTIVE,PRETERIT,PRETERIT_INDICATIVE,TE_FORM,VOLITIONAL|}_$1)",
+        "description": "Sender verb"
```

Comment:
I don't think anybody can understand what is a “sender verb”.

Reply:
Then don't just stand there scolding me xD What would you call these things?

Reply:
What about “First person subject verb”?